src/Microsoft.DotNet.Interactive.CSharpProject/Packaging/PackageBase.cs

```diff
@@ -89,13 +90,17 @@ protected async Task DotnetBuildAsync()
         {
             this.CleanObjFolder();
             var projectFile = this.GetProjectFile();
-            var args = $"/bl:{FullBuildBinlogFileName}";
+
+            string tempDirectoryBuildTarget =

```

Comment:
It's not clear from the name (and because the type is `string`) that this intended to be a file path, not a directory path.

Reply:
changed to directoryBuildTargetsFile